src/modules/endpoint/DefaultDataContainer.js

```diff
@@ -82,7 +82,7 @@ class DefaultDataContainer implements DataContainer {
   }
 
   getResourceCache = async (city: string, language: string): Promise<LanguageResourceCacheStateType> => {
-    const context = new DatabaseContext(city, language)
+    const context = new DatabaseContext(city, null)
```

Comment:
The resources should be available in all languages, that is why we have to set the language to null

Reply:
Would it be feasible to store only `LanguageResourceCacheStateType`s instead of the whole `CityResourceCacheStateType`?

That way we could use the Cache and add the language to the Context.

Reply:
Hmm... It's necessary though if we want to have a shared resource cache, since we need to check the resourceCaches of all other languages as well if we want to cleanup some resource.

Reply:
So... feasible yes, but not within this task :sweat_smile: We would need some other strategy for checking for resource-dependency (then maybe globally for all cities)